release-conf.yaml

```diff
@@ -3,3 +3,8 @@ python_versions:
   - 3
 # whether to allow bot to make PRs based on issues
 trigger_on_issue: true
+
+labels:
+  - bot
+  - release-bot
+  - user-cont
```

Comment:
Is there really a need for this much labels? I think we'd be fine with just `bot`

Reply:
I thought about it like this. If repository has more bots that taking care of different things all of them have at least `bot` label. So if you want filter release bot specifically `bot` is not enough - that's why also `release-bot`. 

And `user-cont` is just promotional one :D